README.md

```diff
@@ -3,6 +3,28 @@
 Service for linking [Sam](https://github.com/broadinstitute/sam) User accounts with registered 3rd party services via
 Oauth2. Bond is a Flask application written in Python 3.7 deployed on Google App Engine.
 
+# Swagger/OpenAPI Interface
+
+The only user interface that Bond makes available is a Swagger UI for directly interacting with its endpoints.  The UI 
+can be reached at the path:
+
+`http://{your_host}/api/docs/`
+
+## Authorization 
+For endpoints that require authorization, you will need a Google Access Token that you can generate using the 
+[gcloud](https://cloud.google.com/sdk/gcloud) command:
+
+`gcloud auth print-access-token`
+
+When you "authorize" in the Bond Swagger UI:
+
+1. Click the `Authorize` button at the top right of the Swagger UI
+1. Copy/paste the generated Access Token from the gcloud CLI into `Value` field
```

Comment:
These each have `1.`? Should it be `1.` `2.` and `3.`?

Reply:
that's markdown... the display of markdown will correctly be 1, 2, 3